eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       echo "Check clang-formatting"
       brew install clang-format
       clang-format --version
-      find sdk/ -regex '.*\.\(cpp\|hpp\)' -exec clang-format -style=file -i {} \;
+      find ./sdk \( -iname '*.hpp' -o -iname '*.cpp' \) ! -iname 'json.hpp' -exec clang-format -i {} \;
```

Comment:
This looks familiar :p

Why are we excluding `json.hpp`?

Reply:
The regex that I used before only works on Linux but does nothing on mac. I didn't invest more time on it and just brought what you did on C here.

The json format check takes about 4 minutes. And we don't expect it to keep changing that file multiple times, so, I considered it's better to ignore the formatting on it. I have already added clang format to it.